### PR TITLE
[feat|sde-backend] : First draft new policies changes

### PR DESCRIPTION
**Pull request by ChetanT-System**
### Added
- Draft code changes of Policies.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by sachinargade123 — CHANGES_REQUESTED** [review]
@ChetanT-System  please update copy right in existing file which has been modified

**Comment by ChetanT-System** [comment]
> @ChetanT-System please update copy right in existing file which has been modified

Updated Please check again

**Review by sachinargade123 — APPROVED** [review]
LTGM

**Review by adityagajbhiye9 — APPROVED** [review]
LGTM

**Review by dvasunin — APPROVED** [review]
LGTM

**Comment by adityagajbhiye9** [comment]
merge after #86

File: modules/sde-common/src/main/java/org/eclipse/tractusx/sde/common/mapper/JsonObjectMapper.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /********************************************************************************
- * Copyright (c) 2022, 2023 T-Systems International GmbH
- * Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
+ * Copyright (c) 2022, 2023, 2024 T-Systems International GmbH
```
Comment (almadigabor) [inline modules/sde-common/src/main/java/org/eclipse/tractusx/sde/common/mapper/JsonObjectMapper.java:2]: This copyright header is incorrect again, please fix it as described in previous PRs.

**Review by almadigabor — CHANGES_REQUESTED** [review]
Please have a look at all copyright header year formats and adjust them same as with previous PRs. Also apply these for your future PRs as well.

File: modules/sde-common/src/main/java/org/eclipse/tractusx/sde/common/mapper/JsonObjectMapper.java (hunk 1 above)
Comment (ChetanT-System) [reply modules/sde-common/src/main/java/org/eclipse/tractusx/sde/common/mapper/JsonObjectMapper.java]: Updated all file copyright

**Comment by ChetanT-System** [comment]
> Please have a look at all copyright header year formats and adjust them same as with previous PRs. Also apply these for your future PRs as well.

@almadigabor  I have updated all copy rights issues of PR #87. Once PR #87 merge then only I can resolve the conflict issue of copy right header. same updated for Next PR. Please review and merge this PR.

**Review by almadigabor — APPROVED** [review]
LGTM

**Pull request merged**